docs/guides/contributing.mdx

```diff
@@ -16,14 +16,17 @@ First, you need to install and be familiar the following:
 [Here](https://github.com/git-guides) is a great guide by GitHub on installing
 and getting started with Git.
 
-2. `node` and `npm`
+2. `node` and `pnpm`
 
 [This guide](https://nodejs.org/en/download/package-manager/) will help you
 install `node` and `npm`. The recommended method is using the `n` version
 manager if you are on MacOS or Linux. Make sure you are using the
 [active LTS version](https://github.com/nodejs/Release#release-schedule) of
 Node.
 
+Once node and npm are installed, run `npm install -g pnpm` to install
```

Comment:
We might want to link directly to the `pnpm` install guide: https://pnpm.io/installation.